app/models/node.ts

```diff
@@ -237,6 +241,18 @@ export default class NodeModel extends AbstractNodeModel.extend(Validations, Col
         return NodeType.Generic;
     }
 
+    /**
+     * The type of this node, as a string.
+     */
+    @computed('isRegistration', 'isRoot')
+    get nodeTypeTranslation() {
+        let translationNode = this.isRoot ? 'project' : 'component';
+        if (this.isRegistration) {
+            translationNode = 'registration';
+        }
+        return this.intl.t(`general.${translationNode}`);
+    }
+
     // This is for the title helper, which does its own encoding of unsafe characters
```

Comment:
Simply `// helper to encode unsafe strings` would suffice here.

Reply:
Your description proposed for this getter isn't accurate, as this is describing the getter below, and not a helper